.github/pull_request_template.md

```diff
@@ -2,10 +2,34 @@
 
 Describe in plain language the motivation (bug, feature, etc.) behind the change in this PR and how the included commits address it.
 
-Fixes: #NNN, #NNN, ...
-
 ## Release notes
+<!--
+
+If this PR does not need to be included in the release notes,
+then simply have a bullet point for `none` directly under
+the `Release notes` section, e.g.
+
+* none
+
+Otherwise, change the `### Todo` section to one or more of the following sections
+below. A section must have at least 1 bullet point. You can add multiple
+sections with multiple bullet points if this PR represents multiple release
+note items. See the CONTRIBUTING.md guidelines for more details.
+
+### Features
```

Comment:
could we simplify this to really have just a place for release note and determine whether it's feature/improvement/bugfix from labels for example? It would simplify the process of creating PR a lot I think not needing to copypaste bunch of sections

Reply:
there are situations in this repo where a PR would contain many types of changes, e.g. fixing a bug to enable a feature. therefore, multiple feature/improvement/bugfix labels would be ambiguous. so i think it would be helpful to leave the instructions to tell the PR author how to structure the changelog lines under sections.

however, i can add to the automation script that processes PR descriptions to fall back on using github label to categorize a changelog entry if a changelog section is not specified in the PR description. in that way, the user can choose to just add a single changelog line and the maintainers add a github label.